test/unit/edgeware/enricher.spec.ts

```diff
@@ -782,7 +781,8 @@ describe('Edgeware Event Enricher Filter Tests', () => {
   /** Session Events */
   it('should enrich new-session event', async () => {
     const kind = EventKind.NewSession;
-    let validators : Array<ValidatorId>;
+    let active : Array<ValidatorId>;
```

Comment:
Maybe we should include some test data to ensure full coverage?

Reply:
I have added plenty of data in the new commit.